### PR TITLE
Allow updating vschema

### PR DESCRIPTION
**Pull request by nickvanw**
This is an experimental feature that allows a user to directly update the VSchema for a particular branch.

### CODE REVIEW

**Review by iheanyi — APPROVED** [review]
Should we implement one for routing rules while we're here? 👀

**Comment by nickvanw** [comment]
> Should we implement one for routing rules while we're here? 👀

Not a bad idea! Let me take a look at what we have to make that possible.

**Comment by nickvanw** [comment]
That was easy enough https://github.com/planetscale/planetscale-go/pull/198/commits/30cb2c277701beab0a2538c48c86dc820d86696d

File: planetscale/branches.go [hunk 1]
```diff
@@ -271,6 +300,54 @@ func (d *databaseBranchesService) VSchema(ctx context.Context, vSchemaReq *Branc
...
 
+func (d *databaseBranchesService) UpdateVSchema(ctx context.Context, updateVSchemaReq *UpdateBranchVschemaRequest) (*VSchemaDiff, error) {
+	path := fmt.Sprintf("%s/vschema", databaseBranchAPIPath(updateVSchemaReq.Organization, updateVSchemaReq.Database, updateVSchemaReq.Branch))
+
+	req, err := d.client.newRequest(http.MethodPatch, path, updateVSchemaReq)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating http request")
+	}
+
+	vSchema := &VSchemaDiff{}
```
Comment (dbussink) [inline planetscale/branches.go:311]: @nickvanw I think this type might be named wrong? Is this really a diff?

Comment (nickvanw) [reply planetscale/branches.go:311]: I did not want to change it in this PR, but it is not a diff - I believe the type is named improperly.

**Pull request merged**